cmd/spire-server/cli/bundle/bundle_test.go

```diff
@@ -36,343 +38,581 @@ o+4RGXoiMFJKysw5GK6jODA2MA8GA1UdEwEB/wQFMAMBAf8wIwYDVR0RAQH/BBkw
+
+	allBundlesJWKS = `****************************************
+* spiffe://domain1.test
+****************************************
+{
+    "keys": [
+        {
+            "use": "x509-svid",
+            "kty": "EC",
+            "crv": "P-256",
+            "x": "fK-wKTnKL7KFLM27lqq5DC-bxrVaH6rDV-IcCSEOeL4",
+            "y": "wq-g3TQWxYlV51TCPH030yXsRxvujD4hUUaIQrXk4KI",
+            "x5c": [
+                "MIIBKjCB0aADAgECAgEBMAoGCCqGSM49BAMCMAAwIhgPMDAwMTAxMDEwMDAwMDBaGA85OTk5MTIzMTIzNTk1OVowADBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABHyvsCk5yi+yhSzNu5aquQwvm8a1Wh+qw1fiHAkhDni+wq+g3TQWxYlV51TCPH030yXsRxvujD4hUUaIQrXk4KKjODA2MA8GA1UdEwEB/wQFMAMBAf8wIwYDVR0RAQH/BBkwF4YVc3BpZmZlOi8vZG9tYWluMS50ZXN0MAoGCCqGSM49BAMCA0gAMEUCIA2dO09Xmakw2ekuHKWC4hBhCkpr5qY4bI8YUcXfxg/1AiEA67kMyH7bQnr7OVLUrL+b9ylAdZglS5kKnYigmwDh+/U="
+            ]
+        },
+        {
+            "use": "jwt-svid",
+            "kty": "EC",
+            "kid": "KID",
+            "crv": "P-256",
+            "x": "fK-wKTnKL7KFLM27lqq5DC-bxrVaH6rDV-IcCSEOeL4",
+            "y": "wq-g3TQWxYlV51TCPH030yXsRxvujD4hUUaIQrXk4KI"
+        }
+    ]
 }
 
-type BundleSuite struct {
-	suite.Suite
+****************************************
+* spiffe://domain2.test
+****************************************
+{
+    "keys": [
+        {
+            "use": "x509-svid",
+            "kty": "EC",
+            "crv": "P-256",
+            "x": "HxVuaUnxgi431G5D3g9hqeaQhEbsyQZXmaas7qsUC_c",
+            "y": "SFd_uVlwYNkXrh0219eHUSD4o-4RGXoiMFJKysw5GK4",
+            "x5c": [
+                "MIIBKjCB0aADAgECAgEBMAoGCCqGSM49BAMCMAAwIhgPMDAwMTAxMDEwMDAwMDBaGA85OTk5MTIzMTIzNTk1OVowADBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABB8VbmlJ8YIuN9RuQ94PYanmkIRG7MkGV5mmrO6rFAv3SFd/uVlwYNkXrh0219eHUSD4o+4RGXoiMFJKysw5GK6jODA2MA8GA1UdEwEB/wQFMAMBAf8wIwYDVR0RAQH/BBkwF4YVc3BpZmZlOi8vZG9tYWluMi50ZXN0MAoGCCqGSM49BAMCA0gAMEUCIQDMKwYtq+2ZoNyl4udPj7IMYIGX8yuCNRmh7m3d9tvoDgIgbS26wSwDjngGqdiHHL8fTcggdiIqWtxAqBLFrx8zNS4="
+            ]
+        }
+    ]
+}
+`
+)
 
-	cert1 *x509.Certificate
-	cert2 *x509.Certificate
+type bundleTest struct {
+	cert1    *x509.Certificate
+	cert2    *x509.Certificate
+	key1Pkix []byte
 
 	ds                 *fakedatastore.DataStore
 	registrationClient *fakeregistrationclient.Client
-	stdin              *bytes.Buffer
-	stdout             *bytes.Buffer
-	stderr             *bytes.Buffer
+	testEnv            *env
 
 	showCmd   cli.Command
 	setCmd    cli.Command
 	listCmd   cli.Command
 	deleteCmd cli.Command
 }
 
-func (s *BundleSuite) SetupTest() {
+func setupTest(t *testing.T) *bundleTest {
 	cert1, err := pemutil.ParseCertificate([]byte(cert1PEM))
-	s.Require().NoError(err)
-	s.cert1 = cert1
+	require.NoError(t, err)
 
-	cert2, err := pemutil.ParseCertificate([]byte(cert2PEM))
-	s.Require().NoError(err)
-	s.cert2 = cert2
+	key1Pkix, err := x509.MarshalPKIXPublicKey(cert1.PublicKey)
+	require.NoError(t, err)
 
-	s.stdin = new(bytes.Buffer)
-	s.stdout = new(bytes.Buffer)
-	s.stderr = new(bytes.Buffer)
```

Comment:
nit: the advantage of this was to not have type assertions everywhere you want to check the contents of the buffer...

Reply:
yea, I thought about that, but I was ok with either way